src/lib/features/events/event-store.ts

```diff
@@ -428,6 +432,56 @@ class EventStore implements IEventStore {
 
         events.forEach((e) => this.eventEmitter.emit(e.type, e));
     }
+
+    async setCreatedByUserId(batchSize: number): Promise<void> {
```

Comment:
Maybe it'd be better to extract this method into a different function, same as the other code from the other task to assign user ids. But for this iteration it seems fine